### ISSUE TITLE
[BUG] Can't access data inside a table

### ISSUE THREAD

**Issue by okon3** [exceljs/exceljs#2297]
## 🐛 Bug Report

I'm developing a web app using react and I need to read data inside and Excel Table. I can get the table using `worksheet.getTable()` but then I'm unable to read the data directly from the table and im forced to just cycle the rows of the entire worksheet.
From the documentation it seems that there should be a `rows` property in the table to access the data, but I keep getting errors as it's undefined in my case.

Here is the Excel i'm trying to read: [test.xlsx](https://github.com/exceljs/exceljs/files/11904782/test.xlsx)

Lib version: 4.3.0

## Steps To Reproduce

```javascript
const workbook = new Workbook();
const xlsx = await workbook.xlsx.load(file);
const worksheet = xlsx.getWorksheet('test');
const table = worksheet.getTable('tbl_dictionary');

table.rows <---  undefined
table.table.rows <---  undefined

table.addRow(['test']) <--- throws error Cannot read properties of undefined (reading 'length')

```

## The expected behaviour:

I'm able to loop through table rows using `table.rows` or `table.table.rows`

**Comment by singhxamit21** [comment]
You can use table.getColumn

for example

// Get Column Wrapper for second column
const column = table.getColumn(1);

**Comment by lhousaine** [comment]
@okon3  , yeah, I have already coming across to the same problem working on a professional project. @singhxamit21 using getColumn(1) works for me but it does not present any useful data about the table content as all properties are undefined as specified in the given screen 
![image](https://github.com/exceljs/exceljs/assets/44302703/4327ca30-16d2-447c-b7b5-b437cb6681c0)

**Comment by Walnussbaer** [comment]
I am experiencing the same issue :/

**Comment by hyperliskdev** [comment]
I also cannot grab any meaningful data using this table function.

**Comment by hb-webdev** [comment]
I can't find a single (free) xlsx-reading library for Node.js that can read Cell contents BY TABLE NAME. This is insane to me.
